PathPlanning/RRT/rrt.py

```diff
@@ -235,7 +242,7 @@ def check_collision(node, obstacleList):
             dy_list = [oy - y for y in node.path_y]
             d_list = [dx * dx + dy * dy for (dx, dy) in zip(dx_list, dy_list)]
 
-            if min(d_list) <= size**2:
+            if min(d_list) <= (size+robot_radius)**2:
```

Comment:
Could you create a variable `collision_dist_squared=(size+robot_radius)**2` at before the for loop, then using it here? This is more efficient.

Reply:
The `size` variable is part of the for-loop, so `collision_dist_squared` cannot be computed before the for-loop. I could add `collision_dist_squared=(size+robot_radius)**2` before the if-statement to make the code more readable?

Reply:
Sorry. I checked `collision_check_with_xy`. Please ignore this comment.